calculator/src/App.js

```diff
@@ -1,25 +1,65 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState, useEffect } from 'react';
+
+import Container from 'react-bootstrap/Container'
+import Col from 'react-bootstrap/Col'
+import Row from 'react-bootstrap/Row'
+
+import "./app.css"
 
 function App() {
+
+  const [partyNumber, setPartyNumber] = useState(5)
+  const [fullCheckAmount, setFullCheckAmount] = useState(0)
+  const [tip, setTip] = useState(0)
+
+  const addOnePerson = () => setPartyNumber(partyNumber + 1)
+  const removeOnePerson = () => setPartyNumber(partyNumber - 1)
+
+  const handleCheckChange = e => setFullCheckAmount(e.target.value)
+  const handleTipChange = e => setTip(e.target.value)
+
+  const perPerson = parseFloat(fullCheckAmount) * ((parseFloat(tip) / 100) + 1) / partyNumber
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <div>
+      <Container>
+        <h1>Straight bills, long friendships</h1>
+        <Row className="justify-content-md-center">
+          <Col>
+            <div className="form-wrapper">
+              <form className="input-form">
+                <label>Input total Check</label>
+                <input name="fullCheckAmount" value={fullCheckAmount} onChange={handleCheckChange} type="number" placeholder="input the total check"></input>
+                <br />
+
+                <label>Adjust party size</label>
+                <p>{partyNumber}</p>
+                <button type="button" onClick={addOnePerson} >Add person</button>
+                <button type="button" onClick={removeOnePerson} >Remove person</button>
+                <br />
+                <label>input the tip %</label>
+                <input name="tip" value={tip} onChange={handleTipChange} type="number" placeholder="input the tip"></input>
+              </form>
+            </div>
+
+          </Col>
+
+          <Col>
+            <article>
```

Comment:
I appreciate the humor added in here, I know it's not "code," but it is always nice to add some flare to your projects. Kudos.